### PR TITLE
added extra feature for tracking activities

### PR DESCRIPTION
**Pull request by witty123**
WHAT:
Added an extra chart for tracking last months activities
WHY:
To add more feature sets

### CODE REVIEW

File: app/components/activity-graph/component.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+                    useFixedDomain: false,
+                    useNiceScale: false,
+                    horizontalOff: false,
+                    verticalOff: true,
+                    unzoomEventType: 'dblclick.zoom'
+                }
+            }
+        };
+
+        $scope.data = [
```
Comment (shreedharshetty) [inline app/components/activity-graph/component.js:50]: What is this? Please move them to constant, if this data is not being updated by someone!

File: app/components/activity-graph/component.js [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+                [1472063400000,0],
+                [1472149800000,0],
+                [1472236200000,0],
+                [1472322600000,0],
+                [1472409000000,0] 
+                ]
+            }
+
+        ];
+      console.log("kya hai");
```
Comment (shreedharshetty) [inline app/components/activity-graph/component.js]: Kya hai? Please remove these statements before merging to master

File: app/components/activity-graph/component.js (hunk 1 above)
Comment (witty123) [reply app/components/activity-graph/component.js:50]: Its not being updated right now, but I have just displayed activities for the last month for now in this extra feature. In future if I have to make it more dynamic then I'll have to update this data using a service.

File: app/components/activity-graph/component.js (hunk 2 above)
Comment (witty123) [reply app/components/activity-graph/component.js]: Didn't see that while creating the pull request. Sorry.

**Pull request merged**